PocketKit/Sources/Sync/Operations/SearchList.swift

```diff
@@ -0,0 +1,72 @@
+import Apollo
+import PocketGraph
+import SharedPocketKit
+
+class SearchList: SyncOperation {
```

Comment:
We also have `AsyncOperation` and `RetriableOperation`. Out of curiosity, would it make sense to leverage one of those? But, this implementation seems fine as well.

Reply:
Good point! I think Search fits better as `AsyncOperation` vs `SyncOperation` since now I think about it we would probably want to just rely on server data at that point and not need to wait for our other operations to complete even if the are pushing changes up to the server. I'll make this update. 

I didn't see any specification on the retry front so don't think it's in the scope of search. Do you see a ticket / requirement that I missed, otherwise I'll clarify during standup. We did have a Try Again button previously if the user was offline, but it was removed and the user would just search again if anything.

Reply:
Discussed over zoom and decided to create a separate service without using OperationQueues to avoid complexity